assets/src/edit-story/panels/components/actionButton.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Shouldn't this be with other buttons in edit-story\components\button

Reply:
Moved all components to generalized versions (and used the same form components in document inspector) in 1219224.